src/app/page.tsx

```diff
@@ -26,6 +26,14 @@ export default function Home() {
     }
   }
 
+  const UserSettingActive = () => {
+    if (user) {
+      return <Button onClick={() => null} className="bg-blue-700 text-slate-300">User Settings</Button>
+    } else {
+      return null;
```

Comment:
I believe we still want the user to be able to access the page when not logged in, so delete this line

Reply:
I don't think they should, if you aren't logged in how can you say if you are a teacher or not / what groups your students are in.